README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
-|   prod   |     [/app/static/commit.txt](https://simplereport.gov/app/static/commit.txt)      |     [/api/actuator/info](https://simplereport.gov/api/actuator/info)      | Dispatched on success of `stg` deploy |                               Used by end users                                |
-|   demo   |   [/app/static/commit.txt](https://demo.simplereport.gov/app/static/commit.txt)   |   [/api/actuator/info](https://demo.simplereport.gov/api/actuator/info)   |  Worflow on success of `stg` deploy   |         Used internally to demo the application to potential end users         |
-| training | [/app/static/commit.txt](https://training.simplereport.gov/app/static/commit.txt) | [/api/actuator/info](https://training.simplereport.gov/api/actuator/info) | Dispatched on success of `stg` deploy | Used externally by potential users to get a better understanding of the product |
-|   stg    |   [/app/static/commit.txt](https://stg.simplereport.gov/app/static/commit.txt)    |   [/api/actuator/info](https://stg.simplereport.gov/api/actuator/info)    |            Push to `main`             |  To validate the application work in the cloud and works with prod like data   |
-|   dev    |   [/app/static/commit.txt](https://dev.simplereport.gov/app/static/commit.txt)    |   [/api/actuator/info](https://dev.simplereport.gov/api/actuator/info)    |     [Action](#deploy-with-action)     |           Development testing/PR validation before merging to `main`           |
-|   dev2   |   [/app/static/commit.txt](https://dev2.simplereport.gov/app/static/commit.txt)   |   [/api/actuator/info](https://dev2.simplereport.gov/api/actuator/info)   |     [Action](#deploy-with-action)     |           Development testing/PR validation before merging to `main`           |
-|   dev3   |   [/app/static/commit.txt](https://dev3.simplereport.gov/app/static/commit.txt)   |   [/api/actuator/info](https://dev3.simplereport.gov/api/actuator/info)   |     [Action](#deploy-with-action)     |           Development testing/PR validation before merging to `main`           |
-|   dev4   |   [/app/static/commit.txt](https://dev4.simplereport.gov/app/static/commit.txt)   |   [/api/actuator/info](https://dev4.simplereport.gov/api/actuator/info)   |     [Action](#deploy-with-action)     |           Development testing/PR validation before merging to `main`           |
-|   test   |   [/app/static/commit.txt](https://test.simplereport.gov/app/static/commit.txt)   |   [/api/actuator/info](https://test.simplereport.gov/api/actuator/info)   |            Push to `main`             |                   Automated testing agsinst `main` (e2e/load)                  |
-|  pentest | [/app/static/commit.txt](https://pentest.simplereport.gov/app/static/commit.txt)  | [/api/actuator/info](https://pentest.simplereport.gov/api/actuator/info)  |     [Action](#deploy-with-action)     |                   Penetration testing (no alerting enabled)                    |
-
-## Cloud Architecture
```

Comment:
⚠️ Moved to https://github.com/CDCgov/prime-simplereport/wiki/Cloud-Architecture